Documentation/ApiOverview/ContentSecurityPolicy/Index.rst

```diff
@@ -46,7 +59,184 @@ need to be enabled.
 
 ..  versionchanged:: 13.0
 
-In TYPO3 backend the Content Security Policy is always enforced.
+    In the TYPO3 backend the Content Security Policy is always enforced.
+
+Within the TYPO3 backend, a specific backend module is available to inspect policy
+violations / reports, and there is also a list to see all configured CSP rules,
+see section :ref:`content-security-policy-backend-rules`.
+
+
+..  _content-security-policy-terminology:
+
+Terminology
+===========
+
+This document will use very specific wording that is part of the CSP W3C specification,
+these terms are not "invented" by TYPO3. Since reading the W3C RFC can be very
+intimidating, here are a few key concepts.
+
+..  note::
+    Skip to the section :ref:`content-security-policy-example` to see a "real-life" usage
+    scenario, if you can better understand from actual code examples.
+
+..  _content-security-policy-terminology-directives:
+
+Directives
+----------
+
+*   CSP consists of multiple rules (or "directives"), that are part of a "policy". This
+    policy says, what functionality the site's output is allowed to use.
+
+*   With that, several HTML tags can be controlled, like from which URLs images can be
+    requested from, if and from where iframes are allowed, if and from where JavaScripts
+    are allowed and so on. There is a long list of applicable directives, see
+    `https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy#directives`__
+    with specific identifiers like `default-src`, `img-src` and so on.
+
+*   Each directive may have several "attributes" (like the allowed URLs).
+
+*   Directives may build upon each other, a bit like CSS definitions (`Cascading Style Sheet`)
```

Comment:
That is hard to grasp, had to read it more than once. I know what you mean, the chain, like described here:
https://brotkrueml.dev/presentations/t3cmd25/csp_einfuehrung.html#/14

So, if frame-src is not defined, it falls back to child-src, and falls back to default-src. But if frame-src is defined, it is used, and the sources from default-src and not used. SO, default-src sources have to be repeated if necessary in frame-src.

Maybe explain it with that example.

Reply:
That's a very helpful description, I basically copied it! :-)